Ryujinx/Ui/SwitchSettings.cs

```diff
@@ -266,9 +279,69 @@ private SwitchSettings(Builder builder, HLE.FileSystem.VirtualFileSystem virtual
             }
 
             _listeningForKeypress = false;
+
+            //Setup System Time Spinners
```

Comment:
```suggestion
            // Setup system time spinners
```
And same for other comments!